src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,25 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne
+    @JoinTable(name = "actors_countries",
```

Comment:
```suggestion
    @JoinColumn(name = "country_id")
```

Reply:
When I change these fields, I get the errors, which are according to this link: https://drive.google.com/file/d/1cccSxEN42NQvkvk8Mwlc8fFAS4O2vOsG/view?usp=sharing

Reply:
Got it. Leave as it is